modules/account-lib/test/unit/coin/stx/keyPair.ts

```diff
@@ -4,7 +4,7 @@ import * as testData from '../../../resources/stx/stx';
 import { Stx } from '../../../../src';
 import { TransactionVersion } from '@stacks/transactions';
 
-describe('Stx KeyPair', function() {
+describe('Stx KeyPair', function () {
```

Comment:
No need for the whitespace before the parens

```suggestion
describe('Stx KeyPair', function() {
```